views/standard/spec/columns.js

```diff
@@ -293,6 +293,13 @@ define(['jquery', 'appearanceUtils', 'underscore'], function($, utils, _) {
                    label.created_at, label.user);
                   node.append(icon);
                }
+               if ((label = pull.getLabel('external_block'))) {
+                  var icon = $('<i>').addClass('fa fa-eye-slash externally-blocked');
```

Comment:
Since the icon we want doesn't exist yet, we can't just select it. This creates a new `<i>` tag and then adds the appropriate classes to it. Then we use `node.append(icon)` to place it where we want.